integrations_test.go

```diff
@@ -100,3 +149,72 @@ func TestRockClient_CreateGCSIntegration(t *testing.T) {
 
 	log.Printf("created gcs integration: %s", gcs.GetName())
 }
+
+type IntegrationsSuite struct {
+	suite.Suite
+	IntegrationsApi *MockIntegrationsAPI
+
+	resp MockIntegrationResponse
+}
+
+func (s *IntegrationsSuite) SetupSuite() {
+	os.Setenv(APIKeyEnvironmentVariableName, "api key")
+	os.Setenv(APIServerEnvironmentVariableName, "https://null.nothing")
+
+	s.IntegrationsApi = &MockIntegrationsAPI{}
+	s.resp = MockIntegrationResponse{}
+}
+
+func (s *IntegrationsSuite) TearDownSuite() {
+	os.Setenv(APIKeyEnvironmentVariableName, "")
+	os.Setenv(APIServerEnvironmentVariableName, "")
```

Comment:
This mutates the environment and will break other tests that rely on having the `ROCKSET_APIKEY` available. If you want a fake api key, just use
```
rc, err := rockset.NewClient(rockset.WithAPIKey("fake"), rockset.WithAPIServer("https://null.nothing"))
```

Reply:
Except that doesn't work because the `NewClient` never looks at them.

Reply:
I'll update that code to work.

Reply:
Nevermind, I see how it works now.
Failing though, so I'll debug this.

Reply:
And this one is entirely on me, it works as expected.

Reply:
BTW, `t.SetEnv` is handy for setting env variables for specific test and restoring values afterwards: https://pkg.go.dev/testing#T.Setenv . Not great when running tests in parallel.

Reply:
Yep, I was also using a Setup, TearDown pair to do it.. but really, not using environment values outside of a main function is the real fix.